lib/hayfork/statement.rb

```diff
@@ -74,6 +80,10 @@ def unnest?
       @unnest == true
     end
 
+    def split?
+      @split_on
```

Comment:
Do we need to `!!` this, or are we okay trusting a consumer to treat it as true/falsey? 🤷

Reply:
I think it couldn't hurt to add that.
I initialize it as `nil`, until given a value. So I was treating it as, "were we given anything?"
`!!` would make it return a boolean though if we were debugging, which might be more expected from a `?` method.